storage/client_event_test.go

```diff
@@ -165,7 +165,7 @@ func (ser *storeEventReader) updateCountString() string {
 
 			for _, methodID := range methodIDs {
 				method := proto.Method(methodID)
-				if count, ok := countset[method]; ok {
+				if count, okCount := countset[method]; okCount {
```

Comment:
"ok" could be whitelisted just like "err", but there were few enough of them that I'd rather leave it for now. In general, I agree, the usage pattern for ok is safe enough that we could whitelist it if the check becomes annoying.